src/ray/gcs/gcs_server/gcs_init_data.cc

```diff
@@ -36,6 +35,8 @@ void GcsInitData::AsyncLoad(const EmptyCallback &on_done) {
   AsyncLoadActorTaskSpecTableData(on_load_finished);
 
   AsyncLoadPlacementGroupTableData(on_load_finished);
+
+  latch.wait();
```

Comment:
If you turn the whole function into sync style, why do you need the atomic var? Why not `on_done` after blocking wait on `latch`?

Reply:
ya left atomic there, removed, and removed on done from here it's just directly called after load() in gcs_server